crates/edr_solidity_tests/tests/testdata/default/cheats/ExpectRevert.t.sol

```diff
@@ -80,20 +84,24 @@ contract ExpectRevertTest is DSTest {
         reverter.revertWithMessage("revert");
     }
 
-    function testFailExpectRevertWrongString() public {
+    function testExpectRevertWithEncodedErrorPrefix() public {
+        vm.skip(true);  // TODO: This change has not been backported yet.
```

Comment:
This should be resolved in a future PR.

Reply:
Wdyt about removing the entire test to make it easy to see that the feature is not supported in a code diff?

Reply:
Yeah, I think that will be more obvious when diffing, just as if we had backported before the test was added.